azavea/pwd-fishway
# Fish and Bubble Animations

## Overview
- New fish and bubble animations added to the screensaver and game pages of the application
- Animation enhancement for buttons (they change size when pressed). This doesn't seem to work well on the ELO (appears to be a tap delay) and I can't seem to figure out why.
- Fixed export of the Biologist video (was cut off)
- Various visual design polish fixups
- Removed extra See the Fishway videos not visible in the original app (they don't fit on the ELO screen) after speaking with @dboyer 

Connects #116, connects #88 

### Demo
<img width="1680" alt="Screen Shot 2019-08-28 at 5 03 26 PM" src="https://user-images.githubusercontent.com/5672295/63892532-e7fea680-c9b5-11e9-9aa5-a3b9bfdf788a.png">
<img src="https://user-images.githubusercontent.com/5672295/63945066-bf6fbe80-ca40-11e9-9477-a870d2e817f0.gif">

Removed videos on See the Fishway to better match existing app:
![Screen Shot 2019-08-29 at 10 51 58 AM](https://user-images.githubusercontent.com/5672295/63968968-d4fbdd00-ca6e-11e9-8785-9e446dd5c74d.png)
<img width="1792" alt="Screen Shot 2019-08-29 at 3 08 42 PM" src="https://user-images.githubusercontent.com/5672295/63969016-ed6bf780-ca6e-11e9-82c5-8c27747880b9.png">


## Testing Instructions
* `git pull`
* Note the fish/bubble animations on the screensaver. The animation should fade in once the images have loaded (so, you shouldn't see the images awkwardly appear as they individually load)
* Click the "Dive in!" button (note animation)
* Once you've clicked, you should see the bubbles move and the fish disappear with the screensaver
* On About tab, swipe to see Joe Perillo.
* Watch video—there should be (a) sound, and (b) weird fish animations at the end of the video
* Go to the game page 
* You should see the bubbles move again and the fish appear
